File: examples/node/sendToGroup.ts
import { ArkeselClient } from '@kalourmade/arkesel-sms';

const groupName = process.argv[2];
if (!groupName) {
  console.error('Usage: tsx sendToGroup.ts <group_name>');
  process.exit(1);
}

const client = new ArkeselClient(process.env.ARKESEL_API_KEY!);
await client.sms.sendToGroup({ sender: 'Arkesel', groupName, message: 'Hello group, from the Arkesel Node SDK' });

console.log(`Sent to group '${groupName}'`);

File: examples/python/send_to_group.py
import os
import sys

from kalourmade_arkesel import ArkeselClient

group_name = sys.argv[1] if len(sys.argv) > 1 else sys.exit("Usage: python send_to_group.py <group_name>")
client = ArkeselClient(os.environ["ARKESEL_API_KEY"])

client.sms.send_to_group(sender="Arkesel", group_name=group_name, message="Hello group, from the Arkesel Python SDK")

print(f"Sent to group '{group_name}'")
